tests/Mono.Android-Tests/Runtime-Microsoft.Android.Sdk/Mono.Android.NET-Tests.csproj

```diff
@@ -39,6 +39,8 @@
   <ItemGroup>
     <Compile Remove="..\Resources\Resource.designer.cs" />
     <Compile Include="System\AppContextTests.cs" />
+    <!-- Negotiate authentication support in the AndroidMessageHandler is available only in .NET 7+ -->
+    <Compile Include="..\Xamarin.Android.Net\AndroidMessageHandlerNegotiateAuthenticationTests.cs" />
```

Comment:
I realize that this support is only on .NET 7, but the tests check for support and `Assert.Ignore()` if support is not present. Thus, if `tests/Mono.Android-Tests/Xamarin.Android.Net/AndroidMessageHandlerNegotiateAuthenticationTests.cs` were listed in `tests/Mono.Android-Tests/Mono.Android-Test.Shared.projitems`, we could verify that the tests are properly ignored in an environment that doesn't support NTLM auth ("Classic" Xamarin.Android).

I think that might make me "feel" better…